cdk/cdk-resources/alert-configuration/src/index.ts

```diff
@@ -0,0 +1,1158 @@
+// Generated by cdk-import
+import * as cdk from 'aws-cdk-lib';
+import * as constructs from 'constructs';
+
+/**
+ * Returns and edits the conditions that trigger alerts and how MongoDB Cloud notifies users. This collection remains under revision and may change. Refer to the legacy documentation for this collection in the following link.
+ *
+ * @schema CfnAlertConfigurationProps
+ */
+export interface CfnAlertConfigurationProps {
+  /**
+   * @schema CfnAlertConfigurationProps#ApiKeys
+   */
+  readonly apiKeys?: ApiKeyDefinition;
+
+  /**
```

Comment:
[q] Could you add unit test that uses the construction? [Not-blocking] you can open a follow PR to add them

Reply:
Sure. Let me check the priorities as I would need to add them to 31 resources.